website/cue/reference/components/sinks/base/greptimedb_logs.cue

```diff
@@ -0,0 +1,448 @@
+package metadata
+
+base: components: sinks: greptimedb: configuration: {
+	acknowledgements: {
+		description: """
+			Controls how acknowledgements are handled for this sink.
+
+			See [End-to-end Acknowledgements][e2e_acks] for more information on how event acknowledgement is handled.
+
+			[e2e_acks]: https://vector.dev/docs/about/under-the-hood/architecture/end-to-end-acknowledgements/
+			"""
+		required: false
+		type: object: options: enabled: {
+			description: """
+				Whether or not end-to-end acknowledgements are enabled.
+
+				When enabled for a sink, any source connected to that sink, where the source supports
+				end-to-end acknowledgements as well, waits for events to be acknowledged by **all
+				connected** sinks before acknowledging them at the source.
+
+				Enabling or disabling acknowledgements at the sink level takes precedence over any global
+				[`acknowledgements`][global_acks] configuration.
+
+				[global_acks]: https://vector.dev/docs/reference/configuration/global-options/#acknowledgements
+				"""
+			required: false
+			type: bool: {}
+		}
+	}
+	batch: {
+		description: "Event batching behavior."
+		required:    false
+		type: object: options: {
+			max_bytes: {
+				description: """
+					The maximum size of a batch that is processed by a sink.
+
+					This is based on the uncompressed size of the batched events, before they are
+					serialized/compressed.
+					"""
+				required: false
+				type: uint: unit: "bytes"
+			}
+			max_events: {
+				description: "The maximum size of a batch before it is flushed."
+				required:    false
+				type: uint: {
+					default: 20
+					unit:    "events"
+				}
+			}
+			timeout_secs: {
+				description: "The maximum age of a batch before it is flushed."
+				required:    false
+				type: float: {
+					default: 1.0
+					unit:    "seconds"
+				}
+			}
+		}
+	}
+	compression: {
+		description: """
+			Compression configuration.
+
+			All compression algorithms use the default compression level unless otherwise specified.
+			"""
+		required: false
+		type: string: {
+			default: "gzip"
+			enum: {
+				gzip: """
+					[Gzip][gzip] compression.
+
+					[gzip]: https://www.gzip.org/
+					"""
+				none: "No compression."
+				zstd: """
+					[Zstandard][zstd] compression.
+
+					[zstd]: https://facebook.github.io/zstd/
+					"""
+			}
+		}
+	}
+	extra_params: {
+		description: """
+			Additional parameters to include in the request.
+
+			These parameters are added to the request URL as query parameters.
+			"""
+		required: false
+		type: object: {
+			examples: [{
+				"source": "Vector"
+			}]
+		}
+	}
+	dbname: {
+		description: """
+			The GreptimeDB [database][database] name to connect.
```

Comment:
```suggestion
			The GreptimeDB [database][database] name to connect.
```
Should there be a link here to https://github.com/greptimeteam/greptimedb?

Reply:
Now that he's pointing [here](https://docs.greptime.com/user-guide/concepts/key-concepts#database) I don't think it should be a problem.